src/sidechaindb.cpp

```diff
@@ -71,27 +72,54 @@ bool SidechainDB::AddWTJoin(uint8_t nSidechain, const CTransaction& tx)
         return false;
 
     const Sidechain& s = ValidSidechains[nSidechain];
-    if (Update(nSidechain, s.GetTau(), 0, tx.GetHash())) {
+
+    std::vector<SidechainWTJoinState> vWT;
+
+    SidechainWTJoinState wt;
+    wt.nSidechain = nSidechain;
+    wt.nBlocksLeft = s.GetTau();
+    wt.nWorkScore = 0;
+    wt.wtxid = tx.GetHash();
```

Comment:
Withdrawal, no segwit stuff involved anywhere in Drivechains. It should be renamed to avoid confusion with segwit, how about just txid?